spec/operations/story_operations/create_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+describe StoryOperations::Create do
+  let(:story_params) do
+    { title: 'Test Story', requested_by: user, state: 'unstarted', accepted_at: nil }
+  end
+
+  let!(:membership) { create(:membership) }
+  let(:user)        { membership.user }
+  let(:project)     { membership.project }
+  let(:story)       { project.stories.build(story_params) }
+
+  subject { -> { StoryOperations::Create.new.call(story: story, current_user: user) } }
+
+  context 'with valid params' do
+    it { expect { subject.call }.to change { Story.count } }
+    it { expect { subject.call }.to change { Changeset.count } }
+    it { expect(subject.call.value!).to be_eql Story.last }
+
+    Story::ESTIMABLE_TYPES.each do |story_type|
```

Comment:
I don't think is a good idea having a loop in a test file. If you have some tests that are "the same" you can use `shared_examples` (personally I don't like that either but is a way of doing this without a loop) or define multiples blocks, remember that tests also works as documentation, so is better to have something written repeatedly to have something DRY, what do you think? We can discuss more about this.